daemon/src/maker_cfd.rs

```diff
@@ -19,11 +19,15 @@ use tokio::sync::watch;
 use xtra::prelude::*;
 use xtra::KeepRunning;
 
-pub struct AcceptOrder {
+pub struct Accept {
     pub order_id: OrderId,
 }
 
-pub struct RejectOrder {
+pub struct Reject {
+    pub order_id: OrderId,
```

Comment:
in the same line as commented on other review, I'll be adding another pair of Accept/Reject structs, so it would be easier if this stayed the same.

Reply:
thanks, missed dropping that one here